nomad/job_endpoint_test.go

```diff
@@ -6801,6 +6803,19 @@ func TestJobEndpoint_Dispatch(t *testing.T) {
 			},
 			noEval: true,
 		},
+		{
+			name:             "valid priority",
+			parameterizedJob: d1,
+			dispatchReq:      reqNoInputValidPriority,
+			err:              false,
+			priority:         reqNoInputValidPriority.Priority,
+		},
+		{name: "invalid priority",
```

Comment:
Nitpick: keeping test case text align makes it easier to scan

```suggestion
		{
			name: "invalid priority",
```